package.json

```diff
@@ -22,7 +22,7 @@
     "@expo/package-manager": "^0.0.43",
     "@types/js-yaml": "^4.0.5",
     "@types/prompts": "^2.0.13",
-    "eslint": "^7.25.0",
+    "eslint": "^8.10.0",
```

Comment:
That's needed because:
`expo-module-scripts@^3.0.3` relies on `eslint-config-universe@^11.0.0` which in turn requires a peer dep of `eslint>=8.10`